openstack/keystone/templates/job-migration.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: keystone-migration
-          image: "{{ default "hub.global.cloud.sap" .Values.repository }}/{{ .Values.api.image }}:{{ .Values.api.imageTag }}"
```

Comment:
`repository` is not set anywhere, must be an old leftover - should be `.Values.global.registryAlternateRegion`, too